apps/pwabuilder/src/script/pages/qualification/app-token.ts

```diff
@@ -616,12 +616,17 @@ export class AppToken extends LitElement {
     this.heroBanners.uncovered = !covered;
   }
 
+  // if showAcceptButton is true we are coming from the accept t and c button
+  // if its false its just the link in the qualifications section
   showTandC(showAcceptButton: boolean){
     this.showTerms = true;
-    this.showTermsNoAccept = showAcceptButton;
-    if(showAcceptButton){
-      recordPWABuilderProcessStep("full_terms_and_conditions_clicked", AnalyticsBehavior.ProcessCheckpoint);
-    }
+    this.showTermsNoAccept = !showAcceptButton;
+
+    // if we are showing the accept button we are in the final t and c
+    // if not it must be from the qualifications box.
+    let location = showAcceptButton ? "final" : "qualification";
```

Comment:
should we keep this **full** to maintain the same step name as before

```suggestion
    let location = showAcceptButton ? "full" : "qualification";
```